build-system/pr-check/utils.js

```diff
@@ -36,7 +36,7 @@ const MODULE_OUTPUT_FILE = `amp_module_${ciBuildSha()}.zip`;
 
 const BUILD_OUTPUT_DIRS = 'build/ dist/ dist.3p/';
 const APP_SERVING_DIRS =
-  'dist.tools/ examples/ test/manual/ texst/fixtures/e2e/';
+  'dist.tools/ examples/ test/manual/ test/fixtures/e2e/';
```

Comment:
I noticed that test fixtures reference resources like images using absolute-ish `/examples/..` paths, which don't resolve correctly when the PR Deploy site path has a prefix. Example:
https://storage.googleapis.com/amp-test-website-1/amp_nomodule_503d8f3ea7b58b202977fec4ec799734be307684/test/fixtures/e2e/amp-lightbox/amp-lightbox.html

This file contains an `amp-img` with src `/examples/img/sample.jpg`. Do you think fixture files should be rewritten to use a relative path like `../../../../examples/img/sample.jpg` to ensure compatibility with both tests and PR Deploy Bot results? Is that overkill?

I'll note that files in `examples` and `test/manual` use `googleusercontent.com` full URLs for resources, which e2e test fixtures can't do (network requests during testing no bueno).

Reply:
Yeah. Ideally each deploy would have its own subdomain to make that problem moot. It's useless for most `/examples/` as it is now.